packages/fluent-ui/src/TextareaWidget/TextareaWidget.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { WidgetProps } from "@rjsf/core";
+
+import TextWidget from '../TextWidget';
+
+const TextareaWidget = (props: WidgetProps) => {
+  const uiProps: any = props.options["props"] || {};
+  let options = {
+    ...props.options,
+    "props": {
+      multiline: true,
+      ...uiProps
+    }
+  };
+  // TODO: rows and columns.
```

Comment:
Are you think uiSchema here? Asking because we can support similar options over in Material UI depending on how this shakes out.